### PR TITLE
[RLlib] Fix Batch Norm Model flakeyness

### PR DESCRIPTION
**Pull request by ArturNiederfahrenhorst**
Signed-off-by: Artur Niederfahrenhorst <artur@anyscale.com>

## Why are these changes needed?

Our batch norm model tests have been amongst the flakiest of RLLib tests.
This PR aims to reduce this flakiness.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by ArturNiederfahrenhorst** [comment]
@sven1977 This seems to get rid of flakeyness but might also get rid of testing learning here. Wdyt?

File: rllib/examples/batch_norm_model.py [hunk 1]
```diff
@@ -42,6 +42,12 @@
 parser.add_argument(
     "--stop-reward", type=float, default=150.0, help="Reward at which we stop training."
 )
+parser.add_argument(
+    "--time-total-s",
+    type=float,
+    default=60 * 60,
+    help="Time after which we stop " "training in seconds.",
```
Comment (ArturNiederfahrenhorst) [inline rllib/examples/batch_norm_model.py:49]: typo I'll fix after review 😓

File: rllib/examples/batch_norm_model.py [hunk 2]
```diff
@@ -42,6 +42,12 @@
 parser.add_argument(
     "--stop-reward", type=float, default=150.0, help="Reward at which we stop training."
 )
+parser.add_argument(
+    "--time-total-s",
```
Comment (sven1977) [inline rllib/examples/batch_norm_model.py:46]:
```suggestion
    "--stop-time",
```

File: rllib/examples/batch_norm_model.py (hunk 1 above)
Comment (sven1977) [inline rllib/examples/batch_norm_model.py:49]:
```suggestion
    help="Time (in seconds) after which we stop training.",
```

**Review by sven1977 — APPROVED** [review]
LGTM, let's try this :)
